template/_watchmanconfig

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
Can you bring this file back please?

Reply:
Sure C: Removed it by mistake :P

Reply:
Why do we need an empty watchman config in a regular React Native app?

Reply:
If there is no watchmanconfig file like this, watchman will not be enabled for the project. We want it to be enabled by default if watchman is installed.

Reply:
Could you confirm this with somebody from Watchman team at FB? We don't need that file in projects that use Jest and it uses watchman without issues

Reply:
Good point, it may actually be a thing that we only need at FB given our settings here. In that case, I'm alright with you removing it.